From e12198ae4896589b324fe7f1367523654ed7711c Mon Sep 17 00:00:00 2001
From: dmerejkowsky <dmerejkowsky@aldebaran-robotics.com>
Date: Fri, 2 Dec 2011 16:42:08 +0100
Subject: [PATCH] taglist: remove boring message

---
 plugin/taglist.vim |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/plugin/taglist.vim b/plugin/taglist.vim
index 59901f6..070af47 100644
--- a/plugin/taglist.vim
+++ b/plugin/taglist.vim
@@ -94,8 +94,6 @@ if !exists('loaded_taglist')
         elseif executable('tags')
             let Tlist_Ctags_Cmd = 'tags'
         else
-            echomsg 'Taglist: Exuberant ctags (http://ctags.sf.net) ' .
-                        \ 'not found in PATH. Plugin is not loaded.'
             " Skip loading the plugin
             let loaded_taglist = 'no'
             let &cpo = s:cpo_save
-- 
1.7.4.msysgit.0

